app/helpers/notification_helper.rb

```diff
@@ -0,0 +1,12 @@
+require "nokogiri"
+
+module NotificationHelper
+  def notification_banner_html(message)
+    parsed_message = Nokogiri.parse(message)
```

Comment:
What is nokogiri doing here? And whats the reasoning behinding using nokogiri? 
More for my understanding than anything else as I haven't seen this before

Reply:
I needed a way of telling if the flash message contains any HTML or not. If it doesn't, we can assume it's a one-line message and wrap it in an h3. 

If it does, it's a bit more complex - the GOVuk standard expects the first line of a flash message to be in an h3, and any other lines in paragraphs. 

So I'm using Nokogiri to tell if there's any HTML, by parsing the message and counting its children. If there's HTML, we output the flash message as-is, without wrapping it in an h3. The only wrinkle here is that we will then need any HTML in the flash messages to match the GOVuk standard (h3 for the first line, para for subsequent lines).

I couldn't think of a better way of telling if a flash message is single or multi line!

I did consider just seeing if the string `include?` any tags (e.g. `message.include?("<")`, but that would mean a message like "This value is < that value" would be interpreted as HTML.